verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts

```diff
@@ -2,7 +2,16 @@
 describe('Linelist table', function () {
     beforeEach(() => {
         cy.task('clearDB', {});
-    })
+        cy.request({
```

Comment:
Can you make this a command in cypress/support/commands.ts so that it can be reused in future tests?

Reply:
done